src/model/reference/BookReference.ts

```diff
@@ -25,12 +32,73 @@ export class BookReference extends BackmatterEntity {
     super();
     this.createEntity(data);
   }
+
+  toXml(): Element {
+    const xmlDoc = new DOMImplementation().createDocument(null, null);
+    const xml = xmlDoc.createElement('element-citation');
+    xml.setAttribute('publication-type', 'book');
+
+    const edition = xmlDoc.createElement('edition');
+    edition.appendChild(xmlDoc.createTextNode(this.edition));
+    xml.appendChild(edition);
+
+    xml.appendChild(serializeReferenceContributorsList('editor', this.editors));
+
+    const elocation = xmlDoc.createElement('elocation-id');
```

Comment:
@FAtherden-eLife is `elocation-id` required on book references? Can't see an example within the kitchen sink or xml example here: https://github.com/libero/editor/issues/98 however it does seem to live on journal references

Reply:
Hi @Cooryd, good point, it's not in any of the samples, eLife haven't used it before. If we could keep it in, that would be great though, I think there could be a (rare) use-case for ebooks.